distribution/options.html

```diff
@@ -6,15 +6,18 @@
 	<p>
 		<label>
 			<strong>Personal token</strong> (optional, <a id="personal-token-link" href="https://github.com/settings/tokens/new?description=Refined%20GitHub&scopes=repo" target="_blank">generate one</a>)<br>
-			<!-- placeholder is set to enable use of :placeholder-shown CSS selector -->
-			<input type="text" name="personalToken" spellcheck="false" autocomplete="off" pattern="[\da-f]{40}" placeholder=" ">
+			<div>
+				<!-- placeholder is set to enable use of :placeholder-shown CSS selector -->
+				<input type="text" name="personalToken" spellcheck="false" autocomplete="off" pattern="[\da-f]{40}" placeholder=" ">
+				<div id="validation"></div>
+			</div>
 		</label>
 	</p>
-	<ul>
-		<li>The token enables <a href="https://github.com/sindresorhus/refined-github/search?q=github-helpers/api" target="_blank">some features</a> to read data from public repositories
-		<li>The <code>public_repo</code> scope lets them edit your public repositories
-		<li>The <code>repo</code> scope lets them edit private repositories as well
-		<li>The <code>delete_repo</code> scope is only used by the <code>quick-fork-deletion</code> feature
+	<ul class="invalid">
+		<li style="list-style: none;">The token enables <a href="https://github.com/sindresorhus/refined-github/search?q=github-helpers/api" target="_blank">some features</a> to read data from public repositories
```

Comment:
If it's not a list item, then it shouldn't be a list item.

I had initially included it here to make it one of the items to _check_, but I saw you put it right next to the field. That's ok too

Reply:
> If it's not a list item, then it shouldn't be a list item.

_and shouldn't be aligned like a list item, and shouldn't be part of `ul`_